modules/stitching/include/opencv2/stitching/detail/blenders.hpp

```diff
@@ -145,6 +145,18 @@ class CV_EXPORTS MultiBandBlender : public Blender
 #if defined(HAVE_OPENCV_CUDAARITHM) && defined(HAVE_OPENCV_CUDAWARPING)
     std::vector<cuda::GpuMat> gpu_dst_pyr_laplace_;
     std::vector<cuda::GpuMat> gpu_dst_band_weights_;
+    std::vector<Point> tl_points_;
+    std::vector<cuda::GpuMat> imgs_with_border_;
+    std::vector<std::vector<cuda::GpuMat>> gpu_weight_pyr_gauss_vec_;
+    std::vector<std::vector<cuda::GpuMat>> gpu_src_pyr_laplace_vec_;
+    std::vector<std::vector<cuda::GpuMat>> ups_;
+    cuda::GpuMat gpu_dst_mask_;
+    cuda::GpuMat gpu_mask_;
+    cuda::GpuMat gpu_img_;
+    cuda::GpuMat weight_map_;
+    cuda::GpuMat add_mask_;
+    int feed_idx;
+    int initialized;
```

Comment:
Please use `gpu_` prefix for all of these vars.

Reply:
Done